runtime-image/test/test.ts

```diff
@@ -139,55 +150,86 @@ const CONFIGURATIONS: TestConfig[] = [
   }
 ];
 
-CONFIGURATIONS.forEach(config => {
-  describe(`nodejs-docker: Image ${config.tag}`, () => {
-    const containerName = uuid.v4();
-    it(config.description, function(done) {
-      this.timeout(2 * TIMEOUT);
-      runDocker(config.tag, containerName, PORT, host => {
-        // Wait for the docker container to start
+describe('runtime image', () => {
+  before(async function() {
+    this.timeout(BUILD_TIMEOUT);
+    await buildDocker(ROOT, 'test/nodejs');
+    const dir = path.join(ROOT, 'test', 'definitions', 'base-install-node');
+    await buildDocker(dir, 'test/definitions/base-install-node');
+  });
+
+  CONFIGURATIONS.forEach(config => {
+    describe(`Image ${config.tag}`, () => {
+      let container: Docker.Container|undefined = undefined;
```

Comment:
Nit. It'll be `undefined` if you don't assign anything.

Reply:
Done